src/main/java/core/basesyntax/Application.java

```diff
@@ -2,6 +2,10 @@
 
 public class Application {
     public static void main(String[] args) {
-        // create three balls using class Lottery and print information about them in console
+        Boll[] bolls = new Boll[3];
+        for (int i = 0; i < bolls.length; i++) {
+            bolls[i] = new Lottery().getRandomBoll();
```

Comment:
Don’t create several instances of the Lottery class if you can use only one instance for your purposes